README.md

```diff
@@ -14,9 +14,15 @@ This is a Cloud Foundry CLI plugin to find the application a given hostname/doma
 ## Usage
 
 ```
-$ cf lookup-route <my.domain.com>
+$ cf lookup-route <my.example.com>
 Bound to:
 <org>/<space>/<app>
-$ cf lookup-route <unknown.domain.com>
+
+# use -t to target the org/space containing the route
+$ cf lookup-route -t <my.example.com>
+Bound to:
+<org>/<space>/<app>
```

Comment:
Sorry, one more thing: mind adding a message of "Changed target to <org>/<space>"?